app/components/widgets/forms/ticket-input.js

```diff
@@ -1,8 +1,12 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { gte } from '@ember/object/computed';
 
 export default Component.extend({
-  classNames: ['ui', 'celled', 'stackable', 'grid', 'ticket-row'],
-
+  classNames  : ['ui', 'celled', 'stackable', 'grid', 'ticket-row'],
+  minMaxValid : computed('ticket.minPrice', 'ticket.maxPrice', function() {
+    gte('this.ticket.maxPrice', 'this.ticket.minPrice');
```

Comment:
@mrsaicharan1 did you even test this ? Please don't copy paste code from the documentation without even seeing how it works and how to use it ... 

And also, please test stuff locally ...

Reply:
@niranjan94 Uh-oh. This would be breaking the check in a reckless manner. I didn't mean to push this. Sorry for that. I've properly used the helper now...

Reply:
@mrsaicharan1 not the template helper. I meant this => [`gte`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/gte).

Try not to keep validation logics in the template.

Reply:
Just to be clear, instead of this in the template`{{unless (gte ticket.maxPrice ticket.minPrice) 'error'}}`, we need it in the controller logic right(using gte)?